docs/developer-quickstart.md

```diff
@@ -84,7 +85,18 @@ Execute the following steps to setup a development environment.
 
     > Keep in mind that Azure role assignments may take up to five minutes to propagate.
 
-## Running the Managment API
+1. Register AAD applications with the Microsoft Identity Platform
+
+   > NOTE: this requires domain admin privileges. You may wish to set up a separate Azure AD tenant for this 
+
+   Follow the [Microsoft Docs Quickstart](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) or run the [AAD App Reg script](../scripts/aad-app-reg.sh) to register two apps
+
+   - **TRE API:**  (with three roles `TREAdmin`, `TREWorkspaceOwner` and `TREResearcher`) This app is used to authenticate users to the API - add users to the admin, owner, and researcher roles.
+   - **TRE Swagger UI**: This app is used to allow Swagger login
+   - You will also want to create applications for workspaces (with the roles `WorkspaceResearcher` and `WorkspaceOwner` that govern who can see what workspaces in the API.
```

Comment:
I am not sure, but I do believe we must have the WorkspaceAdmin role too
```suggestion
   - You will also want to create applications for workspaces (with the roles `WorkspaceAdmin`, `WorkspaceResearcher`, and `WorkspaceOwner` that govern who can see what workspaces in the API.
```

Reply:
There is no such thing as a WorkspaceAdmin -- the closest would be a workspace owner - so I am not changing this

https://github.com/microsoft/AzureTRE/blob/develop/docs/user-roles.md